### PR TITLE
Remove case sensitivity regarding preview file extensions

### PR DESCRIPTION
**Pull request by LostDragonist**
That means ".dds" and ".DDS" should be treated the same.

### CODE REVIEW

**Review by Al12rs — APPROVED** [review]

File: src/modinfodialogimages.cpp [hunk 1]
```diff
@@ -254,7 +254,8 @@ void ImagesTab::select(std::size_t i, Visibility v)
 
     ui->imagesPath->setText(QDir::toNativeSeparators(f->path()));
     ui->imagesExplore->setEnabled(true);
-    if (plugin().previewGenerator().previewSupported(QString::fromStdWString(std::filesystem::path(f->path().toStdWString()).extension().wstring()).remove(0,1)))
+    QString extension = QString::fromStdWString(std::filesystem::path(f->path().toStdWString()).extension().wstring()).remove(0,1).toLower();
```
Comment (isanae) [inline src/modinfodialogimages.cpp:257]: `QFileInfo(f->path()).suffix()`?

Comment (LostDragonist) [reply src/modinfodialogimages.cpp]: That works better. Thanks.

**Comment by Holt59** [comment]
I don't think that's the right way to do it because this will compare against `supportExtensions` from the plugins which is not guaranteed to be lowercase. It's probably easier to make the `contains` case-insensitive in `PreviewGenerator`.

**Comment by isanae** [comment]
Or we can add a guarantee that the given extension will always be lowercase. Since all preview plugins use lowercase right now, it's a win.

**Comment by Holt59** [comment]
> 
> 
> Or we can add a guarantee that the given extension will always be lowercase. Since all preview plugins use lowercase right now, it's a win.

That's assuming all people writing preview plugin follow that convention. Or we'd need modify the actual interface to force lower-cases.

**Comment by Holt59** [comment]
I assumed `supportedExtensions` returned a `QStringList`, so it would have been simple to change the `contains` to be case-insensitive... Since it's a `std::set` it's a bit more work so that's probably good for now.

**Pull request merged**